c200000005.lua

```diff
@@ -0,0 +1,259 @@
+--God-Eyes Phantom Dragon (Manga)
+--Script by Rundas
+local s,id=GetID()
+function c200000005.initial_effect(c)
+	c:EnableReviveLimit()
+	Pendulum.AddProcedure(c,true)
+	--spsummon condition
+	local e1=Effect.CreateEffect(c)
+	e1:SetType(EFFECT_TYPE_SINGLE)
+	e1:SetProperty(EFFECT_FLAG_SINGLE_RANGE+EFFECT_FLAG_CANNOT_DISABLE+EFFECT_FLAG_UNCOPYABLE)
+    e1:SetRange(LOCATION_HAND)
+	e1:SetValue(s.sumval)
+	e1:SetCode(EFFECT_SPSUMMON_CONDITION)
+	c:RegisterEffect(e1)
+	--special summon procedure
+	local e2=Effect.CreateEffect(c)
+	e2:SetType(EFFECT_TYPE_FIELD)
+	e2:SetCode(EFFECT_SPSUMMON_PROC)
+	e2:SetProperty(EFFECT_FLAG_CANNOT_DISABLE+EFFECT_FLAG_UNCOPYABLE)
+	e2:SetRange(LOCATION_HAND)
+	e2:SetCondition(s.spcon)
+	e2:SetOperation(s.spop)
+	c:RegisterEffect(e2)
+	--atk change for non direct attack
+	local e3=Effect.CreateEffect(c)
+	e3:SetCategory(CATEGORY_ATKCHANGE)
+	e3:SetType(EFFECT_TYPE_SINGLE+EFFECT_TYPE_TRIGGER_F)
+	e3:SetCode(EVENT_ATTACK_ANNOUNCE)
+	e3:SetLabelObject(e2)
+	e3:SetCondition(s.atkcon1)
+	e3:SetTarget(s.atktg1)
+	e3:SetOperation(s.atkop1)
+	c:RegisterEffect(e3)
+	--atk change for direct attack
+	local e4=Effect.CreateEffect(c)
+	e4:SetCategory(CATEGORY_ATKCHANGE)
+	e4:SetType(EFFECT_TYPE_SINGLE+EFFECT_TYPE_TRIGGER_F)
+	e4:SetCode(EVENT_ATTACK_ANNOUNCE)
+	e4:SetLabelObject(e2)
+	e4:SetCondition(s.atkcon2)
+	e4:SetTarget(s.atktg2)
+	e4:SetOperation(s.atkop2)
+	c:RegisterEffect(e4)
+	--turn skip
+	local e5=Effect.CreateEffect(c)
+	e5:SetType(EFFECT_TYPE_FIELD+EFFECT_TYPE_TRIGGER_F)
+	e5:SetCode(EVENT_ATTACK_DISABLED)
+	e5:SetRange(LOCATION_MZONE)
+	e5:SetCondition(s.skipcon)
+	e5:SetOperation(s.skipop)
+	c:RegisterEffect(e5)
+	--negate
+	local e6=Effect.CreateEffect(c)
+	e6:SetCategory(CATEGORY_NEGATE)
+	e6:SetType(EFFECT_TYPE_QUICK_O)
+	e6:SetCode(EVENT_CHAINING)
+	e6:SetProperty(EFFECT_FLAG_DAMAGE_STEP+EFFECT_FLAG_DAMAGE_CAL)
+	e6:SetRange(LOCATION_MZONE)
+	e6:SetCondition(s.discon)
+	e6:SetCost(s.discost)
+	e6:SetTarget(s.distg)
+	e6:SetOperation(s.disop)
+	c:RegisterEffect(e6)
+end
+
+--special summon condition
+
+function s.sumval(e,tp,eg,ep,ev,re,r,rp)	
+	return e:GetHandler():IsSummonType(SUMMON_TYPE_PENDULUM)
+end
+
+--special summon proc
+
+function s.cfilter(c)
+	return c:GetSequence()<5
+end
+
+function s.pfilter(c)
+	return c:IsType(TYPE_PENDULUM) and c:IsRace(RACE_DRAGON)
+end
+
+function s.spcon(e,c)
+	if c==nil then return true end
+	local tp=c:GetControler()
+	local g=Duel.GetFieldGroup(tp,LOCATION_MZONE,0)
+	local rg=Duel.GetReleaseGroup(tp)
+	return (#g>0 or #rg>0) and g:FilterCount(Card.IsReleasable,nil)==#g and #g>0 and g:IsExists(s.pfilter,1,nil)
+		and g:FilterCount(s.cfilter,nil)+Duel.GetLocationCount(tp,LOCATION_MZONE)>0
+end
+
+function s.matfilter(c,tp)
+	return c:IsRace(RACE_DRAGON) and c:IsType(TYPE_PENDULUM) and Duel.GetReleaseGroup(tp):IsContains(c)
```

Comment:
here tp is not being passed, also the ``Duel.GetReleaseGroup(tp):IsContains(c)`` is redoundant as you're already iterating ReleaseGroup

Reply:
Ok, Changed the filter as well as the spop.